dev/prepare-build-deps-gpu.sh

```diff
@@ -75,6 +75,6 @@ patchelf --set-soname libirng.so $TARGET_DIR/libirng.so
 
 cp $CMPLR_ROOT/linux/compiler/lib/intel64_lin/libimf.so $TARGET_DIR
 cp $CMPLR_ROOT/linux/lib/libOpenCL.so.1 $TARGET_DIR
-cp $CMPLR_ROOT/linux/lib/libsycl.so.5 $TARGET_DIR
+cp $CMPLR_ROOT/linux/lib/libsycl.so.6 $TARGET_DIR
 
```

Comment:
could you add some file existence check and report error for file missing? 
I think we can build a list of files mapping from a specifc oneapi version to a list for files. such as oneapi_2023_1_libs=( libsycl.so.6 libintlc.so.5  ... ), then we just need to add a new version and update the file list when new oneapi come up.